modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -654,10 +654,12 @@ class Iframe extends React.Component {
 					);
 				}),
 				iframeWindow.Liferay.on('screenFlip', () => {
-					this.props.onOpen({
-						iframeWindow,
-						processClose: this.props.processClose,
-					});
+					if (this.props.onOpen) {
+						this.props.onOpen({
+							iframeWindow,
+							processClose: this.props.processClose,
+						});
+					}
```

Comment:
Just curious, would it be better to just not add the `screenFlip` handler at all rather than adding one that's a no-op?

Reply:
Yep, you're right. Nice catch!